### DISCUSSION TITLE
打开首页异常

### DISCUSSION THREAD

**Discussion by GrahamAshley** [horizoncd/horizon#234] [Q&A]
这是我打开horizon-web服务页面截图，谁能告诉我这是咋回事
<img width="1432" alt="image" src="https://github.com/user-attachments/assets/dc3f6dfe-3ede-4c4a-a6c6-e3a920f6a0ad">

**Comment by GrahamAshley** [comment]
这是浏览器控制台
<img width="1439" alt="image" src="https://github.com/user-attachments/assets/e3b3dfdf-6805-4f41-9a64-0c26496526d1">

**Reply by wokwong** [reply]
请问你解决了吗？我有类似问题，访问直接是白板。
